docs(dialog)integration with other components (#2025)
telerik/blazor-docs: components/dialog/overview.md

Predefined Dialogs are styled substitutes to the standard browser dialogs - confirm, alert and prompt. [Read more about the Blazor Predefined Dialogs]({%slug dialog-predefined%}).

> ## Integration with other components

Reviewer: Headings and titles in English are capitalized

```suggestion
## Integration With Other Components
```

> ## Integration with other components

Reviewer: This section is important, but not more than the built-in Dialog features. Consider moving it before or after Events.

> ## Integration With Other Components

Reviewer 2: As the added content seems implementation-specific, I recommend establishing a new page instead of adding it to the Overview.

> You can implement other components (such as [`Filter`]({%slug filter-overview%}), [`CheckBox`]({%slug checkbox-overview%})) as `DialogContent`. In this case, the two-way binding for the `Value` of these components, will not work as expected. The Dialog needs to [`Refresh`](#dialog-reference-and-methods) to reflect UI changes. You can invoke the Dialog's `Refresh` method:

Reviewer: The information applies to any component and any parameter, is the description should sound more general.

```suggestion
You can use other components inside the `DialogContent`. In this case, the two-way binding for the parameters of these components will not work as expected. The Dialog needs to [`Refresh`](#dialog-reference-and-methods) to reflect UI changes. You can invoke the Dialog's `Refresh` method:
```

> * When using two-way binding - in the `OnChange` event of the component, used as `DialogContent`.

Reviewer 2: Replace the hashes with em dashes in all list items. See the [Punctuation](https://docs.telerik.com/style-guide/punctuation#lists) topic.

> * When using two-way binding - in the `OnChange` event of the component, used as `DialogContent`.

Reviewer: ```suggestion
* When using two-way binding - in the `OnChange` event of the component, used as `DialogContent`.
```
